### ISSUE TITLE
MkdnMoveSource results in error

### ISSUE THREAD

**Issue by aidanlw505** [jakewvincent/mkdnflow.nvim#87]
When trying to move a file and rename a link either by pressing F2 or running MkdnMoveSource I get the following error:

```

Error executing Lua callback: .../share/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/paths.lua:4

31: attempt to perform arithmetic on upvalue 'path_row' (a boolean value)
stack traceback:
        .../share/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/paths.lua:431: in function 'on_confirm'
        /usr/share/nvim/runtime/lua/vim/ui.lua:93: in function 'input'
        .../share/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/paths.lua:421: in function 'confirm_and_execute
'
        .../share/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/paths.lua:505: in function 'on_confirm'
        /usr/share/nvim/runtime/lua/vim/ui.lua:93: in function 'input'
        .../share/nvim/plugged/mkdnflow.nvim/lua/mkdnflow/paths.lua:472: in function 'moveSource'
        ...cal/share/nvim/plugged/mkdnflow.nvim/plugin/mkdnflow.lua:35: in function <...cal/share/nvim/pl
ugged/mkdnflow.nvim/plugin/mkdnflow.lua:35>
```
It is successful in moving the file but it doesn't rename the link.

Here is my mkdwnflow setup.
```
require('git').setup()
require('mkdnflow').setup({
modules = {
    bib = true,
    buffers = true,
    conceal = true,
    cursor = true,
    folds = true,
    links = true,
    lists = true,
    maps = true,
    paths = true,
    tables = true
    },
filetypes = {md = true, rmd = true, markdown = true},
create_dirs = true,             
perspective = {
    priority = 'root',
    fallback = 'first',
    root_tell = ".wiki_root",
    nvim_wd_heel = true
    },    
wrap = true,
bib = {
    default_path = nil,
    find_in_root = true
    },
silent = false,
links = {
    style = 'markdown',
    conceal = true,
    implicit_extension = nil,
    transform_implicit = false,
    transform_explicit = function(text)
    if text:match("/.*/Books/") then 
        text = text:gsub("/.*/Books/", "file:../Books/", 1)
    else 
        text = text:gsub(" ", "-")
        text = text:lower()
        end
        return(text)
        end
        },
    to_do = {
        symbols = {' ', '-', 'X'},
        update_parents = true,
        not_started = ' ',
        in_progress = '-',
        complete = 'X'
        },
    tables = {
        trim_whitespace = true,
        format_on_move = true
        },
    mappings = {
        MkdnEnter = {{'n', 'v', 'i'}, '<CR>'},
        MkdnTab = false,
        MkdnSTab = false,
        MkdnNextLink = {'n', '<Tab>'},
        MkdnPrevLink = {'n', '<S-Tab>'},
        MkdnNextHeading = {'n', ']]'},
        MkdnPrevHeading = {'n', '[['},
        MkdnGoBack = {'n', '<BS>'},
        MkdnGoForward = {'n', '<Del>'},
        MkdnDestroyLink = {'n', '<M-CR>'},
        MkdnMoveSource = {'n', '<F2>'},
        MkdnYankAnchorLink = {'n', 'ya'},
        MkdnYankFileAnchorLink = {'n', 'yfa'},
        MkdnIncreaseHeading = {'n', '+'},
        MkdnDecreaseHeading = {'n', '-'},
        MkdnToggleToDo = {{'n', 'v'}, '<C-Space>'},
        MkdnNewListItem = false,
        MkdnNewListItemBelowInsert = {'n', 'o'},
        MkdnNewListItemAboveInsert = {'n', 'O'},
        MkdnExtendList = false,
        MkdnUpdateNumbering = {'n', '<leader>nn'},
        MkdnTableNextCell = {'i', '<Tab>'},
        MkdnTablePrevCell = {'i', '<S-Tab>'},
        MkdnTableNextRow = false,
        MkdnTablePrevRow = {'i', '<M-CR>'},
        MkdnTableNewRowBelow = {'n', '<leader>ir'},
        MkdnTableNewRowAbove = {'n', '<leader>iR'},
        MkdnTableNewColAfter = {'n', '<leader>ic'},
        MkdnTableNewColBefore = {'n', '<leader>iC'},
        MkdnFoldSection = {'n', '<leader>f'},
        MkdnUnfoldSection = {'n', '<leader>F'}
        }
    })
```

**Comment by jakewvincent** [comment]
Hi @aidanlw505, thanks for reporting this. I just noticed this issue myself a couple days ago. I'll take a look ASAP.

**Comment by jakewvincent** [comment]
This should be fixed now (commit 9ccf1e76c9ddb288bf715594972c9e5093001762). Please pull the update @aidanlw505 and double check that it works as expected. Closing. Thanks again!
